tests/transports-governance/test_runner_hierarchical_comprehensive.py

```
@@ -0,0 +1,830 @@
+    def setup_environment(self) -> bool:
+        """Setup test environment and verify connectivity"""
+        try:
+            # Test basic connectivity
+            health_response = self.governance_client.get_health_check()
+            if health_response.status_code != 200:
+                print(f"❌ Health check failed: {health_response.status_code}")
+                return False
+
+            # Record initial state
+            self.initial_state = {
+                "timestamp": datetime.now().isoformat(),
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)